dotCMS/src/integration-test/java/com/dotcms/contenttype/business/FieldAPITest.java

```diff
@@ -0,0 +1,685 @@
+            assertEquals(list.get(0).value(), fv.value());
+            assertEquals(list.get(1).key(), fg.key());
+            assertEquals(list.get(1).value(), fg.value());
+
+        } finally {
+
+            if (UtilMethods.isSet(contentType) && UtilMethods.isSet(contentType.id())) {
+                contentTypeAPI.delete(contentType);
+            }
+        }
+    }
+
+    @Test
+    public void testSaveOneSidedRelationshipField_returns_newRelationshipObject()
+            throws DotSecurityException, DotDataException {
+        final long time = System.currentTimeMillis();
+        ContentType parentContentType = null;
+        ContentType childContentType  = null;
+        try {
+            parentContentType = ContentTypeBuilder.builder(SimpleContentType.class).folder(
+                    FolderAPI.SYSTEM_FOLDER).host(Host.SYSTEM_HOST).name("parentContentType" + time)
+                    .owner(user.getUserId()).build();
+
+            parentContentType = contentTypeAPI.save(parentContentType);
+
+            childContentType = ContentTypeBuilder.builder(SimpleContentType.class).folder(
+                    FolderAPI.SYSTEM_FOLDER).host(Host.SYSTEM_HOST).name("childContentType" + time)
+                    .owner(user.getUserId()).build();
+
+            childContentType = contentTypeAPI.save(childContentType);
+
+            Field field = FieldBuilder.builder(RelationshipField.class).name("newRel")
+                    .contentTypeId(parentContentType.id()).values("1")
+                    .relationType(childContentType.variable()).build();
+
+            //One side of the relationship is set parentContentType --> childContentType
+            field = fieldAPI.save(field, user);
+
+            final String fullFieldVar =
+                    parentContentType.variable() + StringPool.PERIOD + field.variable();
+
+            final Relationship relationship = relationshipAPI.byTypeValue(fullFieldVar);
+
+            assertNotNull(relationship);
+
+            assertNull(relationship.getParentRelationName());
+            assertEquals(parentContentType.id(), relationship.getParentStructureInode());
+            assertEquals(childContentType.id(), relationship.getChildStructureInode());
+            assertEquals(field.variable(), relationship.getChildRelationName());
+            assertEquals(1, relationship.getCardinality());
+            assertEquals(fullFieldVar, relationship.getRelationTypeValue());
+        } finally {
+            if (UtilMethods.isSet(parentContentType) && UtilMethods.isSet(parentContentType.id())) {
+                contentTypeAPI.delete(parentContentType);
+            }
+
+            if (UtilMethods.isSet(childContentType) && UtilMethods.isSet(childContentType.id())) {
+                contentTypeAPI.delete(childContentType);
+            }
+        }
+    }
+
+    @Test
+    public void testSaveRelationshipField_when_newFieldIsSetToAnExistingRelationship()
```

Comment:
Let's include the expected result of the test? something like `testSaveRelationshipField_When_newFieldIsSetToAnExistingRelationship_SaveShouldSucceed`